regimes/mx/scenarios.go

```diff
@@ -0,0 +1,388 @@
+		},
+	},
+	{
+		Key: TagUse.With(TagTelephoneComms),
+		Name: i18n.String{
+			i18n.EN: "Telephone communications",
+			i18n.ES: "Comunicaciones telefónicas",
+		},
+	},
+	{
+		Key: TagUse.With(TagSatelliteComms),
+		Name: i18n.String{
+			i18n.EN: "Satellite communications",
+			i18n.ES: "Comunicaciones satelitales",
+		},
+	},
+	{
+		Key: TagUse.With(TagOtherMachinery),
+		Name: i18n.String{
+			i18n.EN: "Other machinery and equipment",
+			i18n.ES: "Otra maquinaria y equipo",
+		},
+	},
+	{
+		Key: TagUse.With(TagMedicalExpenses),
+		Name: i18n.String{
+			i18n.EN: "Medical and dental fees and hospital expenses",
+			i18n.ES: "Honorarios médicos, dentales y gastos hospitalarios",
+		},
+	},
+	{
+		Key: TagUse.With(TagMedicalExpenses).With(TagDisability),
+		Name: i18n.String{
+			i18n.EN: "Medical expenses for disability or incapacity",
+			i18n.ES: "Gastos médicos por incapacidad o discapacidad",
+		},
+	},
+	{
+		Key: TagUse.With(TagFuneralExpenses),
+		Name: i18n.String{
+			i18n.EN: "Funeral expenses",
+			i18n.ES: "Gastos funerales",
+		},
+	},
+	{
+		Key: TagUse.With(TagDonation),
+		Name: i18n.String{
+			i18n.EN: "Donations",
+			i18n.ES: "Donativos",
+		},
+	},
+	{
+		Key: TagUse.With(TagMortgageInterest),
+		Name: i18n.String{
+			i18n.EN: "Interest actually paid on mortgage loans (housing)",
+			i18n.ES: "Intereses reales efectivamente pagados por créditos hipotecarios (casa habitación)",
+		},
+	},
+	{
+		Key: TagUse.With(TagSARContribution),
+		Name: i18n.String{
+			i18n.EN: "Voluntary contributions to the SAR",
+			i18n.ES: "Aportaciones voluntarias al SAR",
+		},
+	},
+	{
+		Key: TagUse.With(TagMedicalInsurance),
+		Name: i18n.String{
+			i18n.EN: "Medical insurance premiums",
+			i18n.ES: "Primas por seguros de gastos médicos",
+		},
+	},
+	{
+		Key: TagUse.With(TagSchoolTransportation),
+		Name: i18n.String{
+			i18n.EN: "Mandatory school transportation expenses",
+			i18n.ES: "Gastos de transportación escolar obligatoria",
+		},
+	},
+	{
+		Key: TagUse.With(TagSavingsDeposit),
+		Name: i18n.String{
+			i18n.EN: "Deposits in savings accounts, pension plans premiums",
+			i18n.ES: "Depósitos en cuentas para el ahorro, primas que tengan como base planes de pensiones",
+		},
+	},
+	{
+		Key: TagUse.With(TagSchoolFees),
+		Name: i18n.String{
+			i18n.EN: "Payments for educational services (school fees)",
+			i18n.ES: "Pagos por servicios educativos (colegiaturas)",
+		},
+	},
+	{
+		Key: TagUse.With(TagNoTaxEffects),
+		Name: i18n.String{
+			i18n.EN: "Without tax effects",
+			i18n.ES: "Sin efectos fiscales",
+		},
+	},
+	{
+		Key: TagUse.With(TagSuplementaryPayment),
+		Name: i18n.String{
+			i18n.EN: "Payments",
+			i18n.ES: "Pagos",
+		},
+	},
+	{
+		Key: TagUse.With(TagPayroll),
+		Name: i18n.String{
+			i18n.EN: "Payroll",
+			i18n.ES: "Nómina",
+		},
+	},
+}
+
+var scenarios = []*tax.ScenarioSet{
+	invoiceScenarios,
+}
+
+var invoiceScenarios = &tax.ScenarioSet{
```

Comment:
Is there an opportunity for a default scenario? like "General Expenses"? We could be both explicit and have a default.

Reply:
I always got mixed feelings about defaults. Particularly when they are implicit (i.e. invisible). While they may be convenient, my concern is the user may forget to override the default scenario (no validation error will remind them about it) when that's necessary. 

In other regimes, we're using scenarios mostly for setting subtypes of documents. In that case, there's a natural hierarchy of subtypes and it makes sense to have a default scenario associated with the default invoice type (which is calculated if missing) and no tags. That is not the case here, though.

That's my point of view, but I trust your instinct better than mine on this point. Let me know what you think.

Reply:
Its a tough one! There is a strong argument for saying that GOBL deals with the most common use-case: Digital goods. We can chat about this later if you have time.

Reply:
@samlown I'm merging this PR to move forward with other changes that depend on these. But whether we want to set a default here or not is still open to discussion. I'm taking a note to ensure we chat about it whenever we have a chance.